06_discord_bot/06_discord_bot.md

```diff
@@ -0,0 +1,242 @@
+---
+title: Créer son premier Bot pour discord en python
+date: 2020
+author:
+- Gabrielle Pauvert
+---
+
+Introduction
+============
+
+Dans ce TD, nous allons apprendre pas à pas comment créer un bot pour Discord en python et lui donner quelques premières commandes très simples, par exemple répondre à une commande par un “hello world”.
+
+Création du Bot
+===============
+
+Tout d’abord, connectez vous sur le site de gestion des bots discord :
+<https://discordapp.com/developers/docs/intro>.
+
+Puis cliquez sur l’onglet `Applications` en haut à gauche.
+
+[comment]: ![image](ims/im1.png)
+<center>
+<img src="ims/im1.png" alt="centered image" width="500"/>
+</center>
+
+Si vous ne possédez pas de compte discord, vous devez en créer un car cette action vous demandera de vous authentifier pour vous connecter à votre compte discord. Cliquez sur le bouton `New application` en haut à droite et entrez le nom de votre Bot pour le créer. Vous arrivez sur la page de votre bot, qui contient entre autres son identifiant (`ID`) dont vous aurez besoin pour la prochaine étape. Notez le quelque part, il servira !
+
+[comment]: ![image](ims/im2.png)
+<center>
+<img src="ims/im2.png" alt="centered image" width="500"/>
+</center>
+
+Cliquez ensuite sur l’onglet Bot à gauche puis sur `Add Bot`. Ici, vous
+verrez les informations de votre Bot, dont son pseudo (`username`) et son avatar que vous pourrez modifier selon vos envies, et surtout son token qui **doit toujours rester secret !** (sauf pour vous). Si votre token fuite (sur
+internet par exemple), changez le ! Gardez le dans un coin, on va aussi en
```

Comment:
This could benefit from a footnote, explaining that anyone with the secret token can reply instead of our bot